tests/go_client/testcases/delete_test.go

```diff
@@ -123,7 +123,7 @@ func TestDeleteNotExistName(t *testing.T) {
 	cp := hp.NewCreateCollectionParams(hp.Int64Vec)
 	_, schema := hp.CollPrepare.CreateCollection(ctx, t, mc, cp, hp.TNewFieldsOption(), hp.TNewSchemaOption())
 
-	_, errDelete = mc.Delete(ctx, client.NewDeleteOption(schema.CollectionName).WithPartition("aaa"))
+	_, errDelete = mc.Delete(ctx, client.NewDeleteOption(schema.CollectionName).WithPartition("aaa").WithExpr("int64 < 10"))
```

Comment:
any reason to modify this test case?

Reply:
Empty exprssion is illegal too, and will raise a different error msg after the change.

After this enhancement, plan is checked before partition names because I group partition name and partition key together.